about.html

```diff
@@ -8,10 +8,10 @@
       <div class="row justify-content-center text-center">
         <div class="col col-lg-10">
           <h2>
-            Real problems and uniquely framed solutions in a real world environment.
+            Real problems meet framed solutions
           </h2>
           <p>
-            Vowpal Wabbit is an open, fast, and efficient online learning system that empowers you to explore, test, and research solutions for a real world impact. Through online learning and community collaboration, you can apply real data sets for meaningful results.
+            Explore online machine learning solutions for fast, efficient, and scalable operations with Vowpal Wabbit.

```

Comment:
this is a change in the value.

Reply:
?

"Online machine learning" is the only keyword combination I see with any relevant search volume and traffic so it's important to be consistent. "Online learning" means too many things to the uninitiated...

I wrap the rest elsewhere on page